devtools-ci-index.yaml

```diff
@@ -4123,6 +4127,7 @@
             test_url: prod-preview.openshift.io
             test_suite: planner-functional-tests
             osio_token: be32035b-cdd2-441b-ab27-93d023feeb1d
+            osio_creds: b310559d-91c5-469c-bb35-fc9b8333e619
```

Comment:
@Raunak1203 can you please explain what is `b310559d-91c5-469c-bb35-fc9b8333e619` ?
`osio_creds` var is specifying secret ID in vault here: https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L1963

and I do not see secret with this name in https://vault.devshift.net/ui/vault/secrets/devtools-osio-ci/list/e2e/